test/drivers/netlify-blobs.test.ts

```diff
@@ -39,6 +39,8 @@ describe("drivers: netlify-blobs", async () => {
       token,
       siteID,
       deployID: "test",
+      // Usually defaulted via the environment; only required in a test environment like this
+      region: "us-east-1",
```

Comment:
Is there a fallback behavior if there is no env & default set?

Reply:
No, it bails. But that should only be possible when running outside of the Netlify platform and outside of the Netlify CLI (where Netlify Blobs wouldn't work anyway... so you wouldn't even get to here), or like I mentioned in the PR description if you're using an old version of the Netlify CLI (<17.21.1).

I'll look into making that error more actionable though. It should ideally tell users to upgrade to netlify-cli@latest if they're running through the CLI, instead of just saying `region` is missing.

Reply:
Done!